dataprofiler/tests/profilers/test_boolean_option.py

```diff
@@ -0,0 +1,90 @@
+import os
+import unittest
+from unittest import mock
+
+import pandas as pd
+
+from dataprofiler import Data, ProfilerOptions, Profiler
+from dataprofiler.profilers.profiler_options import BooleanOption
+from dataprofiler.tests.profilers.test_base_option import TestBaseOption
+
+
+@mock.patch('dataprofiler.profilers.data_labeler_column_profile.'
+            'DataLabelerColumn.update', return_value=None)
+@mock.patch('dataprofiler.profilers.data_labeler_column_profile.DataLabeler')
```

Comment:
are these necessary?

Reply:
I'm not sure - they were included in the original profiler options tests.

Reply:
I don't think they are as we don't use the datalabeler in these tests, however previous tests did have it, e.g. `test_data_labeler`